src/here-xyz.ts

```diff
@@ -1093,19 +976,34 @@ async function showSpace(id: string, options: any) {
 
 program
     .command("delete <id>")
-    .description("delete the XYZ space with the given id")
+    .description("delete the XYZ space or project with the given id")
```

Comment:
please remove project here

Reply:
Have removed reference of projects (show / delete) from here-xyz.ts